platform/darwin/src/MGLStyle.h

```diff
@@ -213,8 +213,11 @@ static const NSInteger MGLStyleDefaultVersion = 9;
  @note Adding the same source instance more than once will result in a
     `MGLRedundantSourceException`. Reusing the same source identifier, even with
     different source instances, will result in a 
-    `MGLRedundantSourceIdentifierException`.
- 
+    `MGLRedundantSourceIdentifierException`. Sources should be added in 
+    `MGLMapViewDelegates`'s `mapView:didFinishLoadingStyle:` or
```

Comment:
s/MGLMapViewDelegates/MGLMapViewDelegate/

Also, nowadays we say `-[MGLMapViewDelegate mapView:didFinishLoadingStyle:]` so that jazzy can autolink it: realm/jazzy#362.